### DISCUSSION TITLE
d1 mini esp32-wroom no rf24 found

### DISCUSSION THREAD

**Discussion by melmager** [tbnobody/OpenDTU#1427] [Q&A] [answered]
i have a own PCB , extra 10 uF Cap 
but no rf24 detect - i try 2 rf24 Boards.
CPU bord is d1 mini 32 - esp-wroom chip, detect chip esp32-D0WD-v3
Same PCB works whit d1 mini 8266 and Ahoy
pin settings : clk 18, cs 17, en 5, irq 16, miso 19, mosi 23 

any Ideas ?

**Comment by tbnobody** [comment]
What does Settings --> Hardware show?

**Reply by melmager** [reply, marked as answer]
cange en to 17 and cs to 5 -
works
find out opendtu needs reboot for change the pins - don't work on the fly in my case
my fault sorry .... waste of 2 days :-) 
ok learn how to build :-)
so CE = EN = 17
CSN = CS = 5
